api/lib/notifications.js

```diff
@@ -220,6 +220,7 @@ const startNotificationService = () => {
       const notificationResponse =
         response.data.insert_notifications.returning[0]
       const notification = {
+        id: getMessageTitle(notificationResponse),
```

Comment:
a title is not a good id. get the actual id of the notification from the db?

Reply:
changed